Readme.md

```diff
@@ -713,7 +713,27 @@ The criteria the `partition-bam` tool can be run on can be obtained by passing i
 
 Generating samples (genomes)
 ----------------------------
-Mitty also has features to generate simulated genomes in the form of VCF files. 
+Mitty also has features to generate simulated genomes in the form of VCF files.
+
+### Sampled genomes
+
+The `sample-genome` command generates a new VCF with random genotypes from a given VCF.
+
+```
+@click.option('--vcf', type=click.Path(exists=True))
+@click.option('--output-name', type=str)
```

Comment:
Please use click.Path type so that we can use stdout too - please see `simulate-variants` for an example